config/config-es.json

```diff
@@ -14,34 +14,58 @@
         ]
     },
     "activityInfo": {
-        "startMorningHabitTitleText": "¡Buenos días! Es hora de tu rutina matutina.",
-        "startEveningHabitTitleText": "¡Buenas noches! Es hora de tu rutina nocturna.",
+        "startMorningHabitTitleText":"Good Morning! It’s time to focus. Here are the habits you need to accomplish before you can use your computer.",
+        "startEveningHabitTitleText":"Good Evening! It’s time to focus. Here are the habits you need to accomplish before you can leave for the day.",
         "startActivityButtonText": "Empieza <TIME> <NAME>",
         "activityFinishedText": "<ACTIVITY> completado!✅",
         "lastActivityFinishedText": "<ACTIVITY> completado!✅Te has ganado tu tiempo de pantalla. 💪",
         "startBreakActivityTitleText": "Es hora de un descanso. Haz <TIME> de <ACTIVITY> para ganar más tiempo de pantalla.",
         "doSomeActivityText": "Haz algo de <ACTIVITY>.",
-        "beforeYouStartText": "Antes de comenzar, tómate unos momentos para reflexionar sobre por qué desea adoptar estos hábitos. Luego visualízate comenzando la secuencia del hábito. Repase mentalmente cualquiera de los hábitos que le resulten más difíciles. Este proceso ayudará a reducir la \"fricción límbica\", es decir, preparará tu cerebro para hacer los hábitos sin importar nada, incluso si no tienes la aplicación disponible."
+        "beforeYouStartText": "Antes de comenzar, tómate unos momentos para reflexionar sobre por qué desea adoptar estos hábitos. Luego visualízate comenzando la secuencia del hábito. Repase mentalmente cualquiera de los hábitos que le resulten más difíciles. Este proceso ayudará a reducir la \"fricción límbica\", es decir, preparará tu cerebro para hacer los hábitos sin importar nada, incluso si no tienes la aplicación disponible.",
+        "beforeYouBeginText":"Before you begin, take 30 seconds to visualise yourself performing each of the habits. This sounds kooky but research indicates that doing a mental rehearsal like this increases the likelihood of you completing the habits and makes the habits more sticky so that even if you don't have access to your computer/this app, you'll still do all the habits",
+        "habitsProgressText":"You have completed <duration> of habits - <totalduration> to go. You can do it!",
+        "habitsCompleteText":"You have completed <duration> of habits.",
+        "whatComesNextButtonText":"What comes next?",
+        "buttonGetKookyText":"Get Kooky",
+        "buttonReviseQuantityText":"Revise Quantity"
```

Comment:
I'm pretty sure we have all of these translation keys already. Can you check the files?